src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -0,0 +1,106 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Rule Name="BuildPropertyPage"
+      Description="General"
+      DisplayName="Build"
+      PageTemplate="generic"
+      xmlns="http://schemas.microsoft.com/build/2009/properties" >
+
+  <Rule.Categories>
+    <Category Name="General"
+              Description="General"
+              DisplayName="General" />
+
+    <Category Name="ErrorsAndWarnings"
+              Description="Errors and warnings"
+              DisplayName="Errors and warnings" />
+
+    <Category Name="Output"
+              Description="Output"
+              DisplayName="Output" />
+  </Rule.Categories>
+
+  <Rule.DataSource>
+    <DataSource Persistence="ProjectFile"
+                SourceOfDefaultValue="AfterContext"
+                HasConfigurationCondition="False" />
```

Comment:
This should be `HasConfigurationCondition="True"` since most of the properties on this page are configuration-specific. For the few that are not configuration-specific, you would tag the individual properties with `HasConfigurationCondition="False"`.

Reply:
Good catch, thanks.